app/server/models/organisation.py

```diff
@@ -81,6 +81,10 @@ def timezone(self, val):
     def country_code(self):
         return self._country_code
 
+    @hybrid_property
+    def country(self):
+        return ISO_COUNTRIES[self._country_code]
```

Comment:
in know we do it when setting the country code, but wondering if we should also validate the dict lookup here so the system doesn't 500 if the country code doesn't exist

Reply:
👍